src/test/kotlin/org/rust/lang/core/resolve/RsStdlibResolveTestEdition2021.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.resolve
+
+import org.rust.MinRustcVersion
+import org.rust.MockEdition
+import org.rust.ProjectDescriptor
+import org.rust.WithStdlibRustProjectDescriptor
+import org.rust.cargo.project.workspace.CargoWorkspace
+
+@MinRustcVersion("1.56.0")
```

Comment:
```suggestion
@MinRustcVersion("1.56.0-nightly")
```
since `1.56.0-nightly` < `1.56.0`